flow_action_components/GetRecordTypeInfobyObject/force-app/main/default/classes/GetRecordTypeInfobyObjectTest.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>48.0</apiVersion>
+    <status>Active</status>
+</ApexClass>
```

Comment:
You definitely want to provide some documentation when you post a new component or action. the standard way to do this is to add a README.md file to your local repo, fill it with documentation, and add it and commit it and push it. I usually document in a blog post and link to the blog post from the README.md page to avoid non-developers having to deal with github.

Reply:
Added information to the readme and a link to the post on UnofficialSF